src/components/program-progress/ProgramListingCard.jsx

```diff
@@ -43,23 +43,28 @@ const ProgramListingCard = ({ program }) => {
     return imageURL;
   };
 
+  const authoringOrganization = (
+    program.authoringOrganizations?.length === 1 && program.authoringOrganizations[0].logoImageUrl
+  ) ? {
+      src: program.authoringOrganizations[0].logoImageUrl,
+      alt: program.authoringOrganizations[0].key,
+    } : undefined;
+
   return (
-    <Card className="mb-4 program-listing-card mr-5" onClick={handleCardClick}>
-      <Card.Img
+    <Card
+      className="mb-4 program-listing-card mr-5"
+      isClickable
+      onClick={handleCardClick}
+    >
+      <Card.ImageCap
         src={getBannerImageURL()}
+        logoSrc={authoringOrganization?.src}
+        logoAlt={authoringOrganization?.alt}
         data-testid="program-banner-image"
         className="program-banner-image"
       />
-      {(program.authoringOrganizations?.length === 1 && program.authoringOrganizations[0].logoImageUrl) && (
-        <div className="partner-logo-wrapper shadow-sm">
-          <img
-            src={program.authoringOrganizations[0].logoImageUrl}
-            className="partner-logo"
-            alt={program.authoringOrganizations[0].key}
-          />
-        </div>
-      )}
-      <Card.Body className="program-card-body">
+
+      <Card.Section className="program-card-body">
         <div>
           <div className="font-weight-light d-flex justify-content-between">
             <div>
```

Comment:
1. I believe the JSX below this line could be treated as a `Card.Header` (title = `program.title`, subtitle = the authoring orgs).
2. nit (optional design debt): the colored circles (each `div.progress-item`) in this card could likely be replaced by the [`Bubble` component](https://paragon-openedx.netlify.app/components/bubble/) in Paragon.

Reply:
Yup, they look good as a Stack of Bubbles:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/2307986/170714619-b21b066f-6a02-4e20-89f5-5c674376c771.png">